ramin/js/basic-syntax-35/ramin.js

```diff
@@ -0,0 +1,164 @@
+// add number a to b
+function add(a, b) {
+    return a+b;
+}
+
+/// ----------------------------------- ////
+
+// flip a coin and log the result (head or tail)
+function flipACoin() {
+    //Math.random() generate a 0<=floating_point_number<1 
+    let number  = Math.random();
+    if(number>=0.5)
+        console.log("head")
+    else
+        console.log("tail");
+
+}
+
+/// ----------------------------------- ////
+
+// concat two strings
+function concat(stringA, stringB) {
+    return stringA + ' ' + stringB;
+}
+
+/// ----------------------------------- ////
+
+// 2^x = ?
+function power2(x) {
+    return Math.pow(x,2);
+}
+
+/// ----------------------------------- ////
+
+// user object with first name and last name, both capitalized
+// e.g. getUser('hello', 'world') is { firstName: 'Hello', lastName: 'World' }
+function getUser(firstName, lastName) {
+
+    let Uper_firstName = firstName[0].toUpperCase() + firstName.slice(1); // if firstName = Ramin => R + amin
+    let Uper_lasttName = lastName[0].toUpperCase() + lastName.slice(1);
+
+    return {
+        firstName:Uper_firstName,lastName:Uper_lasttName
+    }
+}
+
+/// ----------------------------------- ////
+
+// greet someone!
+// e.g. greet('ConnecMent') is "Hello, ConnecMent! Hope you the best."
+function greet(name) {
+    console.log(`Hello, ${name}!`);
+}
+
+/// ----------------------------------- ////
+
+// get array of n random numbers
+function getNRandomNumberss(n) {
+    let arr_num = [];
```

Comment:
Same here. The convention in JS for variable names is camelCase, not snake_case.